src/uct/ib/base/ib_iface.c

```diff
@@ -82,13 +83,21 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
    ucs_offsetof(uct_ib_iface_config_t, tx.cq_moderation), UCS_CONFIG_TYPE_UINT},
 
 #if HAVE_DECL_IBV_EXP_CQ_MODERATION
-  {"TX_CQ_MODERATION_COUNT", "-1",
-   "Number of send WQEs for which event is requested.",
-   ucs_offsetof(uct_ib_iface_config_t, tx.cq_moderation_count), UCS_CONFIG_TYPE_INT},
+  {"TX_EVENT_MOD_COUNT", "0",
+   "Number of send WQEs for which event is requested (0 - disabled).",
```

Comment:
"Number of send completions for which an event would be generated (0 - disabled)."

Reply:
done